src/app/containers/PageHandlers/withOptimizelyProvider/index.jsx

```diff
@@ -19,7 +38,7 @@ const withOptimizelyProvider = (Component, noUserId = false) => {
         isServerSide
         timeout={500}
         user={{
-          id: noUserId ? null : getAtUserId(),
+          id: noUserId ? null : getCknsMvt(),
```

Comment:
Ternaries like this, where the truthy value is on the right, can be harder to read and takes a little more cognitive load to understand. It can be an easy mistake to read this like `if there is no user ID then get the cookie value`.

Making the condition clearer upfront and having the truthy part on the left of the ternary is best practice even if it forres you to add a `!` or make the ternary longer.

Also, I'm  a little confused by the variable `noUserId`. Does it mean there is no user ID? Or does it mean we should not user a user ID. If the former I think the var name should be `!hasUserId` or `!userIdExists`. Also if the former, then is this ternary correct? If no user ID then do nothing? Hard to tell because there is no PR description.

Reply:
Yeah I agree, looking back at it, it's a poor choice of variable wording.

The variable is the former, meaning there is noUserId. This is used for storybook stories and for disabling the ID being used on live. I've reworded this now.